### ISSUE TITLE
getting error message on Webbrowser

### ISSUE THREAD

**Issue by red-gee** [jbreizh/imagePaintingSTA#3]
READ ERROR : FILE NOT FOUND

is the only thing appear on browser...

i do not own an android device to test it on android.

**Comment by jbreizh** [comment]
First, have you upload the data content on LittleFs 

Second, honnestly the webpage is not supported/working anymore and i plan to remove it in future version, but i have other plan now (i just buy an apartement with some work + my regular job). What can be done is to use the source of the android app (it's just a web app) you can find here 
 : https://github.com/jbreizh/ImagePainting/tree/master/android/PhoneGap/www as a local page on your computer. The problem with this is that your browser will block the ajax request because of CORS, so the solution i find for my testing is to launch chromium (on linux) with : chromium --user-data-dir=/tmp/ --disable-web-security to disable CORS. I don't know on other OS sorry, you are my first user.

**Comment by red-gee** [comment]
Yes, I’ve uploaded the content on littleFs, but  the error message appears only.  There is a forked version of yours working correctly, but it has only sta mode and no buttons. 

Maybe it would be possible to program a iOS version of the android apk. My programming skills  are not enough to do that. 

Thank you, for your support.
